openverse_catalog/dags/common/slack.py

```diff
@@ -277,7 +277,9 @@ def send_alert(
     known_failures = Variable.get(
         "silenced_slack_alerts", default_var={}, deserialize_json=True
     )
-    if dag_id in known_failures:
+    if dag_id in known_failures and any(
+        error in text for error in known_failures[dag_id]["errors"]
```

Comment:
I think it might be good to do a case-insensitive comparison here, especially for longer error messages where case may differ (we'll also want new tests to this affect should this change be accepted):
```suggestion
        error.lower() in text.lower() for error in known_failures[dag_id]["errors"]
```

Reply:
Good point! Done ✔️